docs/development-guide.md

```diff
@@ -34,6 +34,9 @@ it is **highly** encouraged to:
 * Fork the [ceph-csi repo](https://github.com/ceph/ceph-csi) on Github.
 * Add your fork as a git remote:
    `$ git remote add fork https://github.com/<your-github-username>/ceph-csi`
+* Set up a pre-commit hook to catch issues locally.
+   `$ pip install pre-commit==2.5.1`
+   `$ pre-commit install`
```

Comment:
how pre-commit will be tested locally? can you add more details?

Reply:
After these two steps, user does not need to do anything. The pre-commit will now automatically pick up the config file and check for the rules next time the developer tries to commit changes. :)

Reply:
just add the line that the pre-commit will be triggered when the developer runs `git commit` and also he need to fix basic linter issues first and then continue with git commit.

Reply:
Thanks ... I will add that too.